docs/connector-development/tutorials/custom-python-connector/6-incremental-reads.md

```diff
@@ -0,0 +1,132 @@
+# Incremental reads
+In this section, we'll add support to read data incrementally. While this is optional, you should implement it whenever possible because reading in incremental mode allows users to save time and money by only reading new data.
+
+We'll first need to implement three new methods on the base stream class
+
+The `cursor_field` property indicates that records produced by the stream have a cursor that can be used to identify it in the timeline.
+```python
+    @property
+    def cursor_field(self) -> Optional[str]:
+        return self._cursor_field
+```
+
+The `get_updated_state` method is used to update the stream's state. We'll set its value to the maximum between the current state's value and the value extracted from the record.
+```python
+    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
```

Comment:
I believe this won't be part of the best practices anymore since RFR won't work with connectors that use `get_updated_state`